### ISSUE TITLE
Add elm-deps as dependencies

### ISSUE THREAD

**Issue by robinheghan** [Skinney/elm-git-install#5]
Git deps might have direct and indirect dependencies which aren't git-dependencies. These should be added to the right places in the application's elm.json.

Not entirely sure how this should work. Do we clone the repos and read their tags, scrape package.elm-lang.org? Needs investigating.

**Comment by MartinSStewart** [comment]
I ran into an issue where I installed a git-dependency but my app failed to compile due to missing all the dependencies for that package. At first I thought this was a bug but if I understand correctly that's what this issue is referring too?

**Comment by robinheghan** [comment]
@MartinSStewart Yes. Not sure how to best approach this issue. I don't think `elm install` supports specifying a version range, if it did it would be a simple matter of iterating through all dependencies in `elm.json` and calling `elm install` on them 🤷‍♂️

**Comment by MartinSStewart** [comment]
What about using [elm-json](https://www.npmjs.com/package/elm-json#example-installing-the-latest-available-2xx-version-of-a-package) instead?

**Comment by robinheghan** [comment]
That could work. I do have limited capacity at the moment however. A PR would be much appreciated if you have the time.

**Comment by MartinSStewart** [comment]
I can take a look later today and see if it's something I can puzzle out.

**Comment by robinheghan** [comment]
Awesome!

**Comment by MartinSStewart** [comment]
Sorry, I won't have time to do this either. Maybe someone else can implement this or maybe I'll give it a shot at a later date.
